src/app/modules/user/user.controller.ts

```diff
@@ -77,7 +77,14 @@ export const handleContactVerifyOtp: RequestHandler<
   try {
     await verifyContactOtp(otp, contact, userId)
   } catch (err) {
-    logger.warn(err.meta ?? err)
+    logger.warn({
+      message: `Error occurred whilst verifying contact OTP for ${userId}`,
```

Comment:
minor quibble but can we just make the log message a constant so that we can consistently search using equality in the logs, since we are including userId in the metadata anyway? i.e. `| filter message = 'Error occurred whilst verifying contact OTP'`. Just cleaner if we can always use equality for all logs

Reply:
Can't we use `like` to filter? This allows us to see dynamic stuff at a glance in the message too. But noted on standardization; will fix